beacon_node/network/src/sync/network_context.rs

```diff
@@ -802,7 +801,6 @@ impl<T: BeaconChainTypes> SyncNetworkContext<T> {
                 self.custody_by_root_requests.insert(requester, request);
                 Ok(LookupRequestResult::RequestSent(req_id))
             }
-            // TODO(das): handle this error properly
```

Comment:
This refers to the fact that custody request errors are "nested" but it's okay. So far they are okay to debug